app/controllers/subscriber_lists_controller.rb

```diff
@@ -54,6 +72,14 @@ def bulk_unsubscribe
 
 private
 
+  def updatable_parameters
+    [:title]
+  end
+
+  def has_updatable_parameters?
+    updatable_parameters.map { |field| params.include?(field) }.any?
```

Comment:
`any?` takes a block, so this would be more efficient as

```ruby
updatable_parameters.any? { |field| params.include?(field) }
```

Reply:
ah great shout i'd forgotten any? can take a block